packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/services/phone_number_formatter.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/services.dart';
+
+/// A [TextInputFormatter] that formats phone numbers based on a given format.
+class PhoneNumberFormatter extends TextInputFormatter {
```

Comment:
Avoid adding project specific code to the rx_bloc_cli

Reply:
Done.